rootex/core/audio/audio_source.h

```diff
@@ -36,6 +48,17 @@ class AudioSource
 	ALuint getSourceID() const;
 	/// Get audio duration in seconds.
 	virtual float getDuration() const = 0;
+
+	/// Set the position and model of the audio source
+	void setPosition(Vector3 position);
```

Comment:
Sets*
Do this correction in following comments too.

Reply:
Take in const Vector3&